### ISSUE TITLE
Can't go into settings on ED

### ISSUE THREAD

**Issue by crzyzenn** [joe27g/EnhancedDiscord#148]
Btw, a big fan of this mod. But I ran into an issue, not sure if this is related to ED but here it is.
When I clicked on settings, Discord crashed prompting me to reload, so I went to console to figure out on ed_settings.js, line 604, SwitchItem.Sizes.DEFAULT - DEFAULT was undefined for some reason. so I switched that with a random number and it worked. 

Line 604 - size: props.mini ? SwitchItem.Sizes.MINI : SwitchItem.Sizes.DEFAULT,
to - size: props.mini ? SwitchItem.Sizes.MINI : 16

If I'm not the only one who ran into this, I hope this gets fixed soon :). For the people who ran into this hope this helps you. Cheers!

**Comment by alsoGAMER** [comment]
Same issue here, but with a different error message in the console:
```
TypeError: Cannot read property 'loadPluginSettings' of undefined
    at loadData (/home/USER/Documents/EnhancedDiscord/dom_shit.js:334)
    at Plugin.generateSettings (/home/USER/Documents/EnhancedDiscord/plugins/account_switcher.js:94)
    at PluginSettings (/home/USER/Documents/EnhancedDiscord/plugins/ed_settings.js:312)
    at _a (3833ed3856c1196040a1.js:78)
    at Ls (3833ed3856c1196040a1.js:78)
    at yu (3833ed3856c1196040a1.js:78)
    at Iu (3833ed3856c1196040a1.js:78)
    at cu (3833ed3856c1196040a1.js:78)
    at 3833ed3856c1196040a1.js:78
    at t.unstable_runWithPriority (3833ed3856c1196040a1.js:86)
    at zi (3833ed3856c1196040a1.js:78)
    at Ji (3833ed3856c1196040a1.js:78)
    at Qi (3833ed3856c1196040a1.js:78)
    at fu (3833ed3856c1196040a1.js:78)
    at Ln (3833ed3856c1196040a1.js:78)
    at HTMLDocument.r (e5961e9f972d2a3667b3.js:16)
```

**Comment by joe27g** [comment]
Was fixed in 3f26e91
